### ISSUE TITLE
A11y_Visual Studio Code Services_Download Insiders_Usability: When we press enter on "See SHA-256 Hashes"button, the table opens up and on using Esc it does not collapse.

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5547]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yUsable;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev4;#Keyboard;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win10
Chrome Version: Version 103.0.5060.134(Official build)(64-bit)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab till "Download" button.
4. Scroll down the page and hit enter on "Insiders build" link.
5. Tab till button "See SHA-256 Hashes" and  press enter on it.
6. The table will appear.
7. Now press Esc key to close the table and check whether it is closing or not.

## Actual:
When we press enter on "See SHA-256 Hashes"button, the table opens up and on using Esc it does not collapse. There is no other key available to close the expanded table.

## Expected:
The table should be closed on pressing Esc key or there should be some other buttons available to close the expanded table. When focus is present on the button to expand the table, the focus should remain on it after expansion so user can press it again to collapse the table.

## User Impact:
Keyboard users will find it difficult to close the table and they will have to navigate through entire table to move to content below it.

## Attachment
![ESCKEY](https://user-images.githubusercontent.com/93374043/183059274-a0c65cea-8eb8-42c8-ad39-a300ac2264cb.gif)

**Comment by gregvanl** [comment]
Won't fix. Work around is for user to refresh the page (Ctrl+R) in their browser.
Closing as no "Resolved" state in GitHub issues.
